react/src/components/ServiceLauncherModal.tsx

```diff
@@ -609,25 +725,36 @@ const ServiceLauncherModal: React.FC<ServiceLauncherProps> = ({
                 >
                   <Switch disabled={!!endpoint}></Switch>
                 </Form.Item>
-                {/* <VFolderTableFromItem /> */}
                 {!endpoint ? (
-                  <Form.Item
-                    name={'vFolderName'}
-                    label={t('session.launcher.ModelStorageToMount')}
-                    rules={[
-                      {
-                        required: true,
-                      },
-                    ]}
-                  >
-                    <VFolderSelect
-                      filter={(vf) =>
-                        vf.usage_mode === 'model' && vf.status === 'ready'
-                      }
-                      autoSelectDefault
-                      disabled={!!endpoint}
-                    />
-                  </Form.Item>
+                  <>
+                    <Flex
+                      direction="column"
+                      gap={'xxs'}
+                      align="stretch"
+                      justify="between"
+                    >
+                      <Form.Item
```

Comment:
Can I ask why we need to wrap using `<Flex>` even if it has only one child?

Reply:
Right. I just rearranged the component but forgot to remove it...